### ISSUE TITLE
Error: "Unable to parse color from string:" with Vanilla Example

### ISSUE THREAD

**Issue by quackes** [Rhosys/openapi-explorer#9]
I tried this tool in a simple example derived from the HTML Vanilla Example:

```html
<!doctype html>
<html>
<head>
    <script type="module" src="https://cdn.skypack.dev/openapi-explorer"></script>
    <!-- Or use a local deployed copy -->
    <!--script type="module" src="node_modules/openapi-explorer/dist/openapi-explorer.min.js"></script-->
</head>
<body>
<openapi-explorer
        spec-url="./api-docs-token.json"
        disable-default-colors="true"
        bg-color="#FFF"
        primary-color="#006C69"
        secondary-color="#CCC"
        text-color="#333"
        nav-bg-color="#FFF"
        nav-hover-text-color ="#555"
        nav-text-color ="#333"
> </openapi-explorer>
</body>
</html>
```

I served the html along side with the api-docs-token.json from a local folder with npx serve.

What is the Problem?

Nice tool btw :)

**Comment by wparad** [comment]
No idea but we've been able to reproduce it. We're looking into it.

**Comment by wparad** [comment]
This is fixed with release `0.1.100`, there was an issue with our dynamic resolution of colors, when they weren't set by a UX framework, the fallback wasn't correctly being applied. It should work as you would expect now:

![image](https://user-images.githubusercontent.com/5056218/114529290-16dc9580-9c4a-11eb-9c53-eca77cc51c24.png)
